controllers/article.controller.js

```diff
@@ -0,0 +1,21 @@
+const {allArticlesById, allArticles} = require('../models/articles.models')
+
+const getArticlesById = (request, response) => {
+    const article_id = parseInt(request.params.article_id)
+    console.log(request.params)
+         allArticlesById(article_id).then((article) => {
+            if(!article) {
+                return response.status(404).send({msg: 'Not Found'})
+            }
+        response.status(200).send({article: article})
+    })    
+    }
```

Comment:
Is there any reason why there is code for task 6, in a pull request for task 5? I know it happens, just be a little more aware of it for future pull requests